Fixes #577: return apoc.nodes.count(['User'])

vga91 opened the pull request:
Fixes #577

jexp commented on docs/asciidoc/modules/ROOT/examples/generated-documentation/apoc.meta.nodes-lite.csv:
```diff
@@ -0,0 +1,2 @@
+¦signature
+¦apoc.meta.nodes.count(nodes = [] :: LIST? OF STRING?, config = {} :: MAP?) :: (INTEGER?)
```
aren't these files generated and should not be checked in? everything in generated documentation.

jexp commented on docs/asciidoc/modules/ROOT/partials/usage/apoc.meta.nodes.count.adoc:
```diff
@@ -0,0 +1,66 @@
+
+.Results
+[opts="header"]
+|===
```
I don't think the implementation is what I expected it to be, I expected one result (row) per label

jexp reviewed (changes requested):
I don't think the implementation is what I expected it to be, I expected one result (row) per label

vga91 commented:
> I don't think the implementation is what I expected it to be, I expected one result (row) per label

@jexp Sorry, I thought I answered 😅
I changed the implementation (1 row per label) and deleted some extra auto-generated files.
